packages/core/src/vector/index.warning.ts

```diff
@@ -1,4 +1,4 @@
-import { MastraVector as BaseMastraVector } from './index';
+import { MastraVector as BaseMastraVector } from './vector';
```

Comment:
i think we do not need this anymore

Reply:
Like we don't need the warning.ts anymore for vector?